docs/scos/dev/guidelines/keeping-a-project-upgradable/supported-extension-scenarios/event-subscribers-registration.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Event subscribers registration
+description: Event subscribers registration extension scenario
+last_updated: Mar 13, 2023
+template: concept-topic-template
+related:
+- title: Plugins registration
+  link: docs/scos/dev/guidelines/keeping-a-project-upgradable/supported-extension-scenarios/event-subscribers-registration.html
+- title: Event subscribers registration
+  link: docs/scos/dev/guidelines/keeping-a-project-upgradable/supported-extension-scenarios/event-subscribers-registration.html
+- title: Modules configuration
+  link: docs/scos/dev/guidelines/keeping-a-project-upgradable/supported-extension-scenarios/modules-configuration.html
+---
+
+## Introduction
+
+Manifests support event subscribers registration in the dependency provider.
+
+Manifests fully support event subscribers registration in collection. Restrictions to the order of the plugins in collection are NOT SUPPORTED. New plugin will be added to the end of the collection.
```

Comment:
if by "Restrictions to the order" you mean "after" and "before", then you can drop it. We do not present those features to the customer. It is only for internal core development.